alembic/versions/2024_05_27_769f8c6595a4_add_sequencing_qc.py

```diff
@@ -0,0 +1,32 @@
+"""Add sequencing QC
+
+Revision ID: 769f8c6595a4
+Revises: ec2db27c06e3
+Create Date: 2024-05-27 10:41:06.062540
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "769f8c6595a4"
+down_revision = "ec2db27c06e3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "case",
+        sa.Column(
+            "sequencing_qc_status",
+            sa.Enum("FAILED", "PASSED", "PENDING", name="sequencingqcstatus"),
+            nullable=True,
+        ),
+    )
```

Comment:
I think we should specify "PENDING" as the default here as well, so that the MYSQL table also has that set.

Reply:
Resolved in https://github.com/Clinical-Genomics/cg/pull/3279/commits/d90df282770028edb0eeac43b4731ef5245bf221